src/lib.rs

```diff
@@ -93,8 +93,10 @@ pub enum PandocOption {
     Smart,
     /// --old-dashes
     OldDashes,
-    /// --base-header-level=NUMBER
+    /// DEPRECATED (replaced by ShiftHeadingLevelBy): --base-header-level=NUMBER
```

Comment:
I think we can use Rust deprecation here

```suggestion
    /// --base-header-level=NUMBER
    #[deprecated(note = "replaced by ShiftHeadingLevelBy")]
```

Reply:
Good idea, done.